CHANGELOG.rst

```diff
@@ -79,6 +79,8 @@ Bugfixes
   gesynchroniseerd kunnen worden.
 * [:taiga-is:`3520`, :pr:`1967`]: Missende labels hersteld, die tijdens de migratie voor sommige
   Prosemirror-velden niet waren gekopieerd.
+* [:taiga-is:`3521`, :pr:`1975`]: Probleem opgelost waar de notificatie en mobiele welkom tekst
```

Comment:
Misschien beter "Probleem opgelost waar de registratie notificatie en welkom tekst in elkaar overlopen op mobiel."?

want deze gebeurt eigenlijk alleen op de Home page, omdat andere pagina's waarschijnlijk geen overlay meer krijgen.